### PR TITLE
Mc sp admin rules for venues

### PR DESCRIPTION
**Pull request by mcon85**
hid edit and delete buttons on a venue unless the member was admin (will need to add check for current user as owner once venues has user_id implemented)
also put in logic on venues controller to not allow access to edit or destroy paths unless user is an admin. 
styled the edit/delete buttons on the venue show page

### CODE REVIEW

File: app/views/venues/show.html.erb [hunk 1]
```diff
@@ -1,15 +1,26 @@
 <div class="venue-details">
   <h1><%= @venue.name %></h1>
-  <%= button_to 'Delete', venue_path(@venue), method: 'delete',
-                                              class: "alert button" %>
-  <%= link_to @venue.website, @venue.website, class: "website",
-                                              target: "_blank" %>
+  <div>
+    <% if current_user && current_user.role == 'admin' %>
```
Comment (nvanselow) [inline app/views/venues/show.html.erb]: Switch this to `if current_user && current_user.admin?`

**Comment by nvanselow** [comment]
I just thought of this when I tried to test out these features on the webpage. I couldn't because I can't create an admin user. Can we create a seeder for admin?

**Comment by mcon85** [comment]
Can hack your account to be an admin  using rails console rather than seeding, for now

**Comment by nvanselow** [comment]
Looks good! Merging.
Tests are dropping because or JS (capybara-webkit) tests are not running on codeship. But those test do run/pass on our local machines.

**Pull request merged**